AuthService/parsistence/interface.go

```diff
@@ -0,0 +1,8 @@
+package parsistence
+
+import "authservice/parsistence/entities"
+
+type JwtPersistence interface {
```

Comment:
Usually interface naming is the same as class naming in terms of meaning, so `JwtPersister` is a better name imho